oandm/sprint-container-image-pulled-manually/sprint-container-image-pulled-manually.md

```diff
@@ -0,0 +1,69 @@
+# How do I generate an alert for detecting changes in Kubernetes Workloads Image Pull Policy and ensuring policy compliance using Logging Analytics?
+
+Duration: 15 minutes
+
+This lab will walk you through the steps to create schedule tasks to detect when a **Container Image is Pulled with the latest Tag**.
+This lab explains how to schedule search tasks to identify if a container image was pulled manually in Kubernetes.
+
+The images pulled by Kubernetes can be found in the Oracle **"Kubernetes Event Object Logs"** out-of-the-box source. These logs can be monitored from the Log Explorer.
+
+1. Navigate to the Log Explorer (Refer to [How to use Log Explorer for analytics and visualizations?](?lab=sprint-log-explorer-for-analytics-and-visualizations) for more info) and run this query:
+
+    ```MQL
+      <copy>
+      'Log Source' = 'Kubernetes Container Status Logs' and Image like '%:latest' | stats count(Image) by Pod, Container
+      </copy>
+    ```
+
+  ![Figure 1: Run Kubernetes Images that has the 'latest' tag query](images/run-kubernetes-images-with-latest-tag-query.png)
+
+  **P.s:** If you can't see any results, this is probably because you still have no Kubernetes logs of containers using images with "latest" tag ingested.
+2. Save the log explorer search by clicking on "Actions" and Selecting "Save As..."
+  ![Figure 2: Save the log search](images/save-log-explorer-search.png)
+3. Fill the name, the description and Click on "Save":
+  ![Figure 3: Fill the form to save the search](images/fill-save-search-form.png)
+4. Create a Scheduled Detection Rule by clicking on "Actions" and Selecting "Create detection rule":
+  ![Figure 4: Click on "Create detection rule" button](images/click-on-detection-rule-creation.png)
+5. Fill the Detection Rule form showing up (Refer to [How to create and trigger an ingest time detection rule?](?lab=sprint-create-and-trigger-ingest-time-detection-rule) for more details):
+  ![Figure 5: Fill the form to create the detection rule](images/fill-detection-rule-form.png)
+
+Prepare the logs to upload and trigger the Detection Rule making its metrics available to the alarms service:
```

Comment:
Use `##` to create a secondary header format

Reply:
Done!